lib/TransferConfigurations.cs

```diff
@@ -48,16 +53,22 @@ public TransferConfigurations()
             this.parallelOperations = Environment.ProcessorCount * 8;
             this.MemoryChunkSize = Constants.DefaultMemoryChunkSize;
 
+#if DOTNET5_4
+            this.maxListingConcurrency = 6;
+#else
+            this.maxListingConcurrency = 2;
+#endif
```

Comment:
Minor: This "this.maxListingConcurrency = 2;" initialization is not necessary as there's default value when creating DirectoryListScheduler..

Reply:
Done